parm/config/config.com

```diff
@@ -0,0 +1,91 @@
+# shellcheck shell=bash
+# Ignore shellcheck warnings about variables not being expanded; this is what we want
+# shellcheck disable=SC2016
+echo "BEGIN: config.com"
+
+# These are just templates. All templates must use single quotations so variable
+#   expansion does not occur when this file is sourced. Substitution happens later
+#   during runtime. It is recommended to use the helper function `generate_com()`,
+#   to do this substitution, which is defined in `ush/preamble.sh`.
+#   
+#   Syntax for generate_com():
+#       generate_com [-rx] $var1[:$tmpl1] [$var2[:$tmpl2]] [...]]
+#
+#       options:
+#           -r: Make variable read-only (same as `decalre -r`)
+#           -x: Mark variable for declare -rx (same as `declare -x`)
+#       var1, var2, etc: Variable names whose values will be generated from a template
+#                   and declared
+#       tmpl1, tmpl2, etc: Specify the template to use (default is "${var}_TMPL")
+#
+#   Examples:
+#       # Current cycle and RUN
+#       YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_ANALYSIS
+#
+#       # Previous cycle and gdas
+#       RUN=${GDUMP} YMD=${gPDY} HH=${gcyc} generate_com -rx \
+#           COM_ATMOS_HISTORY_PREV:COM_ATMOS_HISTORY_TMPL
+#
+#       # Current cycle and COM for first member
+#       MEMDIR='mem001' YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_HISTORY
+#
+
+# 
+# If any restart, input, or analysis template is updated, `setup_expt.py.fill_COMROT_cycled()`
+#   must correspondingly be updated to match.
+# 
+if [[ "${RUN_ENVIR:-emc}" == "nco" ]]; then
+    COM_OBS_TMPL=$(compath.py "${envir}/obsproc/${obsproc_ver}")'/${RUN}.${YMD}/${HH}/atmos'
```

Comment:
Maybe not here right now but do we want to ask OBSPROC to change from `atmos` to `obs` for obs files like we did in global-workflow?